hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -11,11 +11,12 @@
   To enable usage while the device is sleeping, it runs on a slow always-on clock.
   In addition, it has a low power periodic scan mode for monitoring ADC channels.
   '''
+  cip_id:             "1",
   design_spec:        "../doc",
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_adc_ctrl",
-  version:            "1.0",
+  version:            "1.0.0",
```

Comment:
Given that this changes all register offsets in all IPs, should we bump all major versions to `2.0.0`?

@cfrantz @alphan @moidx

Reply:
To stay on the prudent side, and since we have 32bit version fields, I have bumped all versions to `2.0.0`.